bindata/etcd/quorumguard-deployment.yaml

```diff
@@ -72,13 +72,17 @@ spec:
                 - /bin/sh
                 - -c
                 - |
-                  declare -r health_endpoint="https://localhost:2379/health"
-                  declare -r cert="/var/run/secrets/etcd-client/tls.crt"
-                  declare -r key="/var/run/secrets/etcd-client/tls.key"
-                  declare -r cacert="/var/run/configmaps/etcd-ca/ca-bundle.crt"
-                  export NSS_SDB_USE_CACHE=no
+                  set -xe
                   [[ -z $cert || -z $key ]] && exit 1
-                  curl --max-time 2 --silent --cert "${cert//:/\:}" --key "$key" --cacert "$cacert" "$health_endpoint" |grep '{ *"health" *: *"true" *}'
+                  /usr/bin/etcdctl \
+                    --key=/var/run/secrets/etcd-client/tls.key
+                    --cert=/var/run/secrets/etcd-client/tls.crt
+                    --cacert=/var/run/configmaps/etcd-ca/ca-bundle.crt
+                    --command-timeout=2s \
+                    --dial-timeout=2s \
+                    --endpoints=https://loclahost:2379 \
+                    endpoint health -w json | grep \"health\":true
+
```

Comment:
If we are running `etcdctl` should not the image on L52 be `etcdImage` instead of `cliImage`?

https://github.com/openshift/cluster-etcd-operator/blob/3fde37aace23d4d0978b7b104cfabab4140bb77d/bindata/etcd/quorumguard-deployment.yaml#L52

Reply:
yeah thanks this is still a bit raw I will ping you guys when it's ready for review.